src/ruleChecker.ts

```diff
@@ -28,12 +28,31 @@ export default class RuleChecker {
     check: Check,
     findings: Finding[]
   ): Promise<void> {
+    const numScopesChecked = await this.checkScope(
+      appMapFile,
+      appMapIndex,
+      check,
+      check.scope,
+      findings
+    );
+    if (numScopesChecked === 0 && check.scope === 'command') {
```

Comment:
Falling back on root scope seems better than emitting nothing at all from the scope.